angular.json

```diff
@@ -51,6 +51,16 @@
               "node_modules/ace-builds/src-min-noconflict/theme-idle_fingers.js",
               "node_modules/ace-builds/src-min-noconflict/ext-searchbox.js",
               "node_modules/bootstrap/dist/js/bootstrap.js"
+            ],
+            "allowedCommonJsDependencies": [
+              "ts-md5/dist/md5",
+              "bodybuilder",
+              "rxjs/internal/observable/of",
```

Comment:
We shouldn't be using `rxjs/internal`, glad this effort caught it.. Looks like it's only in 1 place, auth.guard.ts, so you should be able to remove this line, and change the import in auth.guard.ts to `import { of } from "rxjs";`